src/watcher/store.ts

```diff
@@ -143,6 +143,10 @@ export class AppointmentStore extends StartStopService implements IAppointmentSt
         return false;
     }
 
+    public getById(id: string) {
```

Comment:
This isnt documented or tested. Also we should specify that it returns Appointment | undefined, since the types dont pick this up...

Reply:
Perhaps we should change `appointmentsById` with a Map?
Also, testing seems kinda redundant. Maybe we could move appointmentsById to a private mAppointmentsById and have a public getter that returns a ReadonlyMap?

Reply:
Sounds fine to me

Reply:
Done in 225e9d34f5174aa0c22919a37d06b65472341d18.